PWGHF/TableProducer/treeCreatorOmegacSt.cxx

```diff
@@ -405,9 +407,11 @@ struct HfTreeCreatorOmegacSt {
                               momenta[0][0], // omega momentum
                               momenta[0][1],
                               momenta[0][2],
+                              trackCasc.sign(),
```

Comment:
The method `sign()` should return a `short` (https://github.com/AliceO2Group/AliceO2/blob/f63f7a183882b899c34e11eddf84e59e7a00836d/Framework/Core/include/Framework/AnalysisDataModel.h#L132C53-L132C53). Did you verify that this fill works well, given that the corresponding column expectes a `bool`? Otherwise you should probably cast this variable into a `bool`.

Reply:
Hi @mfaggin, good catch, I will make this an explicit conversion. Thanks!